### ISSUE TITLE
TI-99 and TI-99/4A support

### ISSUE THREAD

**Issue by rryan22** [TASEmulators/BizHawk#3592]
Can you please add and support to work on 2.9 release? Because It's made by Texus Instruments. Add TI-99 and TI-99/4A support to Bizhawk! Thank you!

**Comment by YoshiRulz** [comment]
The TI-99/4 was an early PC that """competed""" with the likes of the Atari 400 and Commodore VIC-20. Unlike those, it used an in-house CPU, and not the same one as the TI-83 so we can't re-use that either. However, I did find [this existing emulator](https://www.mrousseau.org/programs/ti99sim/) written in C++. There were [at least 235 games](https://www.videogamehouse.net/rarity.txt) developed for the TI-99/4 and TI-99/4A.

**Comment by rryan22** [comment]
Yes! It can run TI-99/4 and TI-99/4A games on Bizhawk. Is there a core of TI-99/4 and TI-994A to support for Bizhawk? Thanks!

**Comment by YoshiRulz** [comment]
I'm not sure what you're trying to say. This is not supported in BizHawk, and it is not planned either.

**Comment by rryan22** [comment]
Yeah, what about Atari 400/800/XEGS/5200 cores instead, but allows you to play Atari 8-Bit/5200 games on Bizhawk. Are they supported?

**Comment by YoshiRulz** [comment]
Not yet, see #3317.

**Comment by getCursorsExe** [comment]
~~I think they just don't have proper savestates. They can't even be waterboxed without modifying waterbox code, which is very volatile to other cores.~~
